app/services/project_search_service.rb

```diff
@@ -25,6 +25,6 @@ def search_by_ukprn(ukprn)
   end
 
   private def ukprn_pattern(query)
-    query.match?(/\d{8}$/)
+    query.match?(/^\d{8}$/)
```

Comment:
I wonder if we should put in more coverage around incorrect query strings like this - clearly originated with me, but might be worthwhile?

Reply:
Nope, sorry, must have missed them!